install-with-linux.md

```diff
@@ -54,10 +54,16 @@ When downloading a release, select the brunch...tar.gz file from the assets at t
 3. Once both files have been downloaded, the Brunch release and your chosen ChromeOS recovery, open a terminal. In most Linux distros you can just press **ctrl + alt + t** to open it quickly.
 4. Make sure that pv, cgpt, tar and unzip are installed.
 
-```sudo apt update && sudo apt -y install pv cgpt tar unzip```
-  
-  * My example uses `apt`, a package manager for Debian and Ubuntu based distros. If you use Arch, you will need [vboot-utils][vboot-utils] for access to cgpt and a different package manager may be needed to install the rest.
+On ubuntu/debian/linux mint<br/>
+```sudo apt update && sudo apt -y install pv cgpt tar unzip```<br/><br/>
+Or on arch:<br/>
+```sudo pacman -Syy && sudo pacman -S pv tar unzip```<br/>
+For cgpt, you have to use a AUR helper like yay or pamac 
 
+```yay -S cgpt```<br/>
```

Comment:
Ideally, you would not want users to use an AUR helper. You should instead instruct them on building it without one. You could also mention that if you already have an AUR helper, you can just use that.